spec/controllers/presentations_controller_spec.rb

```diff
@@ -55,29 +55,12 @@
 
     before { mock_pundit_user_as(non_admin) }
 
-    describe 'GET #new' do
```

Comment:
I think the policy test is good. It tests that the policy returning correctly.
The policy is tested in one context, i.e. `new?` returns `true` is meaningful only if you understand the context of the policy's responsibility, and that you're sure it's correctly applied to the controller.
It might seem intuitive that is `new?` returns `true` on the policy means that the controller is functioning properly, but as the complexity of the application grows, you'll want to test the the policy returns `true` and `false` correctly _and_ that your controller produces the responses (`http_status`, redirect, etc.) that you expect.

Keep in mind what the controller's responsibility is and how it is different than the policy's.